mappings/net/minecraft/tag/ItemTags.mapping

```diff
@@ -1,6 +1,11 @@
 CLASS yz net/minecraft/tag/ItemTags
+	CLASS yz$a AutoUpdatingTag
+		FIELD a modCount I
```

Comment:
`lastModificationId`

Reply:
`expectedModCount`, like java standard library?

Reply:
No. That's very different! `expectedModCount` checks modification counts to detect and throw `ConcurrentModificationException`s. This just keeps track of the last cached modification ID to check if it needs to be updated.